recipes-graphics/westeros/westeros-sink.bbappend

```diff
@@ -1,14 +1,9 @@
-SINK_SOC_PATH = "${@bb.utils.contains('MACHINE_FEATURES', 'vc4graphics', 'v4l2', 'rpi', d)}"
+SINK_SOC ?= ""
```

Comment:
I would use even weaker assignment( ??=). Let's keep that in mind when we will be adding new configuration options

Reply:
done